sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/RateLimitedSamplerTests.cs

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using Azure.Monitor.OpenTelemetry.Exporter.Internals;
+using OpenTelemetry.Trace;
+using Xunit;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Tests;
+
+public class RateLimitedSamplerTests
+{
+    [Fact]
+    public void RespectLocalParentDecision()
+    {
+        var sampler = new RateLimitedSampler(10000);
```

Comment:
```suggestion
        var sampler = new RateLimitedSampler(1000);
        sleep(10 millis); // ensure when sampler sees first trace, it will think rate is <= 1/100
        
        create 100 child spans (no more sleeping)
```